channels/pe.m3u

```diff
@@ -301,5 +313,8 @@ https://5ee0faac3bbae.streamlock.net/visionnoticias/visionnoticias/playlist.m3u8
 https://5ee0faac3bbae.streamlock.net/visionmusica/visionmusica/playlist.m3u8
 #EXTINF:-1 tvg-id="Willax.pe" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/willaxtv/picture?width=320&height=320" group-title="General",Willax (480p)
 https://iptv.dc-team.com/query?url=https://www.dailymotion.com/WillaxTV
+http://free.fullspeed.tv/query?url=https://www.dailymotion.com/WillaxTV
```

Comment:
Quack.

Reply:
Line 315 was not deleted.

Reply:
Sorry because I had resolved conflicts

Reply:
There is no pound sign before EXTINF:-1

Should be #EXTINF:-1